google/cloud/internal/async_polling_loop.cc

```diff
@@ -44,13 +45,14 @@ class AsyncPollingLoopImpl
   future<StatusOr<Operation>> Start(future<StatusOr<Operation>> op) {
     auto self = shared_from_this();
     auto w = WeakFromThis();
-    auto const& options = CurrentOptions();
-    promise_ = promise<StatusOr<Operation>>([w, options]() mutable {
-      if (auto self = w.lock()) {
-        OptionsSpan span(std::move(options));
-        self->DoCancel();
-      }
-    });
+    CallContext call_context;
+    promise_ = promise<StatusOr<Operation>>(
+        [w, c = std::move(call_context)]() mutable {
+          if (auto self = w.lock()) {
+            ScopedCallContext scope(c);
```

Comment:
Move the `CallContext` into the `ScopedCallContext`?

Reply:
Fixed.